src/components/RadioButtonWithLabel.js

```diff
@@ -55,13 +58,17 @@ const RadioButtonWithLabel = (props) => {
                     label={props.label}
                     hasError={props.hasError}
                 />
-                <TouchableOpacity
+                <PressableWithFeedback
+                    focusable={false}
+                    accessible={false}
                     onPress={() => props.onPress()}
+                    onLongPress={() => props.onPress()}
```

Comment:
Same as above. Shouldn't we want radio buttons to have long press functionality in future if required?

Reply:
I made it this way so it would be consistent with present implementation. `TouchableOpacity` will trigger onPress on long pressing, but Pressable in our implementation won't.